src/events/AppPromotionCard.tsx

```diff
@@ -43,53 +42,21 @@ const LinkWrapper = styled.div`
   }
 `
 
-const AppStoreLink = styled.a`
-  text-decoration: none;
-  margin: 5px 0;
-  ${mediaQueries.md} {
-    margin-right: 0;
-  }
-`
-const AppStoreImg = styled.img`
-  background: black;
-  width: 140px;
-  height: 48px;
-  object-fit: contain;
-`
-export const title = 'Download our app'
-export const description = `Download our iOS & Android app to carry our interactive Parade map,
-stage line up info and afterparty information around with you on the day`
+export const title = 'List your event'
+export const description = `Whether you're hosting an Instagram Live Q&A, digital drag brunch or a theatre run, we'd love to showcase your event on our platform.`
 
 const AppPromotionCard: React.FC = () => {
```

Comment:
Since this is no longer an `AppPromotionCard`, this should probably be renamed (along with the file and tests) 🙂